pkgs/tools/networking/ferm/default.nix

```
@@ -1,12 +1,12 @@
 { stdenv, fetchurl, makeWrapper, perl, ebtables, ipset, iptables, nixosTests }:
 
 stdenv.mkDerivation rec {
-  version = "2.4.1";
+  version = "2.5.1";
   pname = "ferm";
 
   src = fetchurl {
-    url = "http://ferm.foo-projects.org/download/2.4/ferm-${version}.tar.xz";
-    sha256 = "1fv8wk513yysp4q0i65rl2m0hg2lxwwgk9ppprsca1xcxrdpsvwa";
+    url = "http://ferm.foo-projects.org/download/2.5/ferm-${version}.tar.xz";
```

to future proof this
```suggestion
    url = "http://ferm.foo-projects.org/download/${lib.versions.majorMinor version}/ferm-${version}.tar.xz";
```